app/assets/javascripts/geoblacklight/viewers/iiif.js

```diff
@@ -21,5 +21,5 @@ GeoBlacklight.Viewer.Iiif = GeoBlacklight.Viewer.extend({
 
     // expand viewer element
     $(this.element).parent().attr('class', 'col-md-12');
-  }
+  },
```

Comment:
I was using an airbnb style linter, and kinda waffled back and forth on these. I can remove them for the sake of compatibility, but as of January, [IE8 is no longer supported by Microsoft](https://support.microsoft.com/en-us/lifecycle#gp/msl-ie-dotnet-an). Have any browser metrics for Earthworks or NYU GeoBL?